### PR TITLE
Add code for TC02 (Log in with blank username)

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Write code for case login with blank username.

### CODE REVIEW

File: src/test/java/tests/LoginTests.java [hunk 1]
```diff
@@ -25,4 +26,20 @@ public void tc001_LoginWithValidUsernameAndPassword() {
...
         Assert.assertTrue(loginPage.isLogoutTabDisplayed(), "User can not log in successfully with valid account");
     }
+
+    @Test(testName = "User can not log in to Railway with blank username")
+    public void tc002_LoginWithBlankUsername() {
+        LogHelper.info("Go to Login Tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Log in with with blank username");
+        loginPage.login("", DataHelper.getRandomPassword(15));
```
Comment (anhlvt) [inline src/test/java/tests/LoginTests.java:36]: don't need to random password here. You can input any

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -25,4 +26,20 @@ public void tc001_LoginWithValidUsernameAndPassword() {
...
+
+        LogHelper.info("Verify: Is error message displayed as expected");
+        Assert.assertEquals(
+                loginPage.getErrorMessage(),
```
Comment (anhlvt) [inline src/test/java/tests/LoginTests.java:40]: I think no need to break line here.

File: src/test/java/tests/LoginTests.java (hunk 1 above)
Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java:36]: I don't understand? At first, I get password from Constant.PASSWORD. However, I do not believe it is for the general public so I call method random password.

Comment (anhlvt) [reply src/test/java/tests/LoginTests.java:36]: I mean you can input any character to password field. No need to use random here bz it do not affect you test result each time you run.

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java:36]: Yeah Okie

File: src/test/java/tests/LoginTests.java (hunk 2 above)
Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java]: Thank you, Anh :) I fixed it. Please check again.

File: src/test/java/tests/LoginTests.java [hunk 3]
```diff
@@ -21,8 +22,21 @@ public void tc001_LoginWithValidUsernameAndPassword() {
...
+
+    @Test(testName = "User can not log in to Railway with blank username")
+    public void tc002_LoginWithBlankUsername() {
+        LogHelper.info("Go to Login Tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Log in with blank username");
+        loginPage.login("", Constants.PASSWORD);
+
+        LogHelper.info("Verify: Is error message displayed as expected");
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:38]: this log should be something like `Verify error message displayed....`

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java:38]: Yeah Okie, I got it

**Review by anhlvt — APPROVED** [review]

File: src/test/java/tests/LoginTests.java (hunk 3 above)
Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java]: I fixed it, please check again

**Review by tanmle — APPROVED** [review]

**Pull request merged**